Pluralize.NET/Pluralizer.cs

```diff
@@ -1,89 +1,6 @@
-﻿using Pluralize.NET.Rules;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text.RegularExpressions;
-
-namespace Pluralize.NET
+﻿namespace Pluralize.NET
 {
-    public class Pluralizer
+    public class Pluralizer : PluralizerBase, IPluralize
```

Comment:
Any reason why we need a PluralizerBase? Pluralizer now looks to be an empty shell class.
I am not very familiar with this pattern, can we discuss?

Reply:
Currently its an empty shell, what I was originally thinking and wanted to see how you wanted to handle, is making a definitions file to separate the "language definitions" from the code.  

Whether its json, xml, flat file etc, we could have a default set of definitions, but someone could load a custom set.  Meanwhile, all the core logic is in the `PluralizerBase` making it easy to extend/override implementation.  I see it as more of an advanced feature, but if later we want to support multi-language then this could prove useful.

What are your thoughts here?

Reply:
Ok. Infact I am not keen on supporting multiple languages as scope of the original library is also only English language. If any other language has to be supported, I think that the implementation could be totally different from current, and inheritance might do more harm than good in that case.

However let's keep these changes, as I don't see why not as of now, and we'll cross the multi-language bridge when it comes.

Please go ahead with merging this, if you are also on the same page.

Reply:
Now that you mention it, I agree, different language support may be more complicated.  We'll leave it as is for now, since the versioning model is major version, we can count on a newer version being considered a breaking change going forward.

I'll move forward with this one.